app/src/main/java/org/mozilla/focus/MainActivity.kt

```diff
@@ -93,8 +94,14 @@ class MainActivity : LocaleAwareAppCompatActivity(), OnUrlEnteredListener {
         TelemetryWrapper.stopSession(this)
     }
 
+    override fun onStart() {
+        super.onStart()
+        Pocket.startBackgroundUpdates()
+    }
+
     override fun onStop() {
         super.onStop()
+        Pocket.stopBackgroundUpdates() // Don't regularly hit the network in the background.
```

Comment:
In general since this is a tv with dedicated network whose primary use case is watching video content, I think background network is probably okay.